week-1/K-functions-parameters/exercise3.js

```diff
@@ -1,9 +1,14 @@
-// Write your function here
+// IT'S WORKING BUT I THINK THERE IS A BETTER WAY
+var name = "Daniel";
+
+function createGreeting() {
```

Comment:
Here, you want to have a function that has a parameter. So `function createGreeting() {` should be `function createGreeting(name) {`. This means you don't need the line above `var name = "Daniel";`